webservices/cxf/src/resources/applicationContext.xml

```diff
@@ -160,7 +160,9 @@ http://cxf.apache.org/schemas/jaxws.xsd">
     <bean id="org.sakaiproject.webservices.SakaiJob" class="org.sakaiproject.webservices.SakaiJob" init-method="init" parent="org.sakaiproject.webservices.AbstractWebService" />
     <bean id="org.sakaiproject.webservices.SakaiLogin" class="org.sakaiproject.webservices.SakaiLogin" init-method="init" parent="org.sakaiproject.webservices.AbstractWebService" />
     <bean id="org.sakaiproject.webservices.SakaiPortalLogin" class="org.sakaiproject.webservices.SakaiPortalLogin" init-method="init" parent="org.sakaiproject.webservices.AbstractWebService" />
-    <bean id="org.sakaiproject.webservices.SakaiScript" class="org.sakaiproject.webservices.SakaiScript" init-method="init" parent="org.sakaiproject.webservices.AbstractWebService" />
+    <bean id="org.sakaiproject.webservices.SakaiScript" class="org.sakaiproject.webservices.SakaiScript" init-method="init" parent="org.sakaiproject.webservices.AbstractWebService" >
+        <property name="sqlService" ref="org.sakaiproject.db.api.SqlService"/>
+    </bean>
```

Comment:
Yes, I'll do, so If we add this in the Abstract and remove here, (as you say, it is the logical thing), we should remove that too in the SakaiReport one, that has the same addition.  I think that should be a different ticket anyway (I mean remove the reference to sqlService in the SakaiReport).